### DISCUSSION TITLE
TS0502B Device Not Supported

### DISCUSSION THREAD

**Discussion by dyde121** [Koenkk/zigbee2mqtt#12419] [Devices]
Running latest dev build, this device is showing not supported although TS0502B should be recognised.

https://www.ikuu.com.au/downloadable-assets/product-specifications/MIDS000010Z%20Data%20Sheet.pdf
https://www.zigbee2mqtt.io/devices/TS0502B.html

Zigbee Model TS0502B
Zigbee Manufacturer _TZ3000_armwcncd

**Comment by Koenkk** [comment]
Added.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by dyde121** [reply]
Working! Thanks!
